AIDevGallery/Samples/BaseSamplePage.cs

```diff
@@ -42,4 +61,38 @@ internal void SendSampleInteractedEvent(string? customInfo = null)
     {
         SampleParams?.SendSampleInteractionEvent(customInfo);
     }
+
+    internal async void ShowException(Exception ex, string? optionalMessage = null)
+    {
+        var msg = optionalMessage ?? ex switch
+        {
+            COMException
+                when ex.Message.Contains("the rpc server is unavailable", StringComparison.CurrentCultureIgnoreCase) =>
+                    "The WCL is in an unstable state.\nRebooting the machine will restart the WCL.",
+            _ => $"Error:\n{ex.Message}{(optionalMessage != null ? "\n" + optionalMessage : string.Empty)}"
+        };
+        ContentDialog exceptionDialog = new()
```

Comment:
We should have a copy exception details button here

Reply:
This is added now.